csuntechlab/iSTART
# STR-94 -- BE: Create email for students when they finish a module

# Description

There are some templates already in the project

TASK IS DONE WHEN:

An email template (blade file) exists that can be sent out to a student in regards to them having finished a module. Requires contract and service for email, but not calling the service
  
# Testing

1. Log in as any participant
2. visit `/sendCompleteModuleMailAdminTest` and ensure the admin email still works
3. visit `/sendCompleteModuleMailStudentTest` and ensure the student email is now working
